components/common/src/main/java/com/hotels/styx/common/format/DefaultHttpMessageFormatter.java

```diff
@@ -0,0 +1,44 @@
+/*
+  Copyright (C) 2013-2019 Expedia Inc.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+ */
+package com.hotels.styx.common.format;
+
+import com.hotels.styx.api.HttpRequest;
+import com.hotels.styx.api.HttpResponse;
+import com.hotels.styx.api.LiveHttpRequest;
+import com.hotels.styx.api.LiveHttpResponse;
+
+/**
+ * Provides default formatting for requests and responses.
+ */
+public class DefaultHttpMessageFormatter implements HttpMessageFormatter {
+
+    public String formatRequest(HttpRequest request) {
+        return request == null ? null : request.toString();
+    }
+
+    public String formatRequest(LiveHttpRequest request) {
+        return request == null ? null : request.toString();
```

Comment:
This code is okay for this PR. 

However it might be safer to return an empty string, or "NA" etc instead of bare `null` when request is not provided. What do you think?

Reply:
I wasn't sure what circumstances this would occur in (if ever?), so I left the behaviour as already was.  It's no problem changing it though even in this PR.

Reply:
Ok

Reply:
changed to N/A

Reply:
Now changed to string "null" instead of "N/A".  It's more meaningful and is consistent with what we have in HttpRequestMessageLogger.